packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-node-ciemss.vue

```diff
@@ -273,11 +274,24 @@ watch(
 			`;
 			const summaryResponse = await createLLMSummary(prompt);
 
-			const portLabel = props.node.inputs[0].label;
+			const parentModel = await getModelByModelConfigurationId(modelConfigId.value as string);
+			const calibratedModelConfig: ModelConfiguration = {
+				name: 'Calibrated configuration',
+				description: 'Calibrated configuration',
+				simulationId: state.calibrationId,
```

Comment:
Given that we have two uses for the word Simulation (Object + forecast)
and that this is always expected to be a calibration's run id, is there a reason why we are not calling it calibrationId?

Reply:
The "new" model configuration object is a bit vague - it has both simulatonId and calibrationId, I asked which one should be used for calibrated-model-configs, I got back "simulationId" ... CC @blanchco